ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/EventsTableTreeEvents.scala

```diff
@@ -86,10 +89,10 @@ private[events] trait EventsTableTreeEvents { this: EventsTable =>
   ).mkString(", ")
 
   private val witnessesAggregation =
-    "array_agg(event_witness) as event_witnesses"
+    "tree_event_witnesses"
 
   private val treeEventsTable =
-    "participant_events natural join participant_event_transaction_tree_witnesses"
+    "participant_events"
```

Comment:
These can also be folded in the query itself at this point. They were meant to be short-hands for longer fragments.